packages/core/src/index.ts

```diff
@@ -140,7 +140,6 @@ export * from "./hooks/useUniqueId";
 export * from "./hooks/useWidth";
 
 export * from "./providers/Provider";
-export * from "./providers/ThemeProvider";
```

Comment:
this is a breaking change

Reply:
I know, but is there even any use case where the ThemeProvider is being used instead of the Provider?